docs/hadoop.adoc

```diff
@@ -28,68 +83,59 @@ gremlin> :plugin use tinkerpop.hadoop
-gremlin> if (!hdfs.exists('data/grateful-dead.kryo')) hdfs.copyFromLocal('data/grateful-dead.kryo','data/grateful-dead.kryo')
+gremlin> if (!hdfs.exists('/grateful-dead.kryo')) hdfs.copyFromLocal('data/grateful-dead.kryo','/grateful-dead.kryo')
 ==>null
-gremlin> graph = GraphFactory.open('conf/hadoop-graph/hadoop-load.properties')
+gremlin> graph = GraphFactory.open('conf/hadoop-graph/hadoop-bulk-load.properties')
 ==>hadoopgraph[gryoinputformat->nulloutputformat]
-gremlin> blvp = BulkLoaderVertexProgram.build().writeGraph('conf/janusgraph-cql.properties').create(graph)
+gremlin> blvp = BulkLoaderVertexProgram.build().writeGraph('conf/janusgraph-cassandra-bulk-load.properties').create(graph)
 ==>BulkLoaderVertexProgram[bulkLoader=IncrementalBulkLoader,vertexIdProperty=bulkLoader.vertex.id,userSuppliedIds=false,keepOriginalIds=true,batchSize=0]
 gremlin> graph.compute(SparkGraphComputer).program(blvp).submit().get()
 ...
 ==>result[hadoopgraph[gryoinputformat->nulloutputformat],memory[size:0]]
 gremlin> graph.close()
 ==>null
-gremlin> graph = GraphFactory.open('conf/hadoop-graph/read-cassandra-3.properties')
-==>hadoopgraph[cassandrainputformat->gryooutputformat]
-gremlin> g = graph.traversal().withComputer(SparkGraphComputer)
-==>graphtraversalsource[hadoopgraph[cassandrainputformat->gryooutputformat], sparkgraphcomputer]
+gremlin> graph = JanusGraphFactory.open('conf/janusgraph-cassandra-bulk-load.properties')
+==>standardjanusgraph[cql:[127.0.0.1]]
+gremlin> g = graph.traversal()
+==>graphtraversalsource[standardjanusgraph[cassandrathrift:[127.0.0.1]], standard]
 gremlin> g.V().count()
-...
 ==>808
+gremlin> g.E().count()
+==> 8046
 ----
 
 [source, properties]
 ----
-# hadoop-load.properties
-
+# hadoop-bulk-load.properties
 #
 # Hadoop Graph Configuration
 #
 gremlin.graph=org.apache.tinkerpop.gremlin.hadoop.structure.HadoopGraph
-gremlin.hadoop.graphInputFormat=org.apache.tinkerpop.gremlin.hadoop.structure.io.gryo.GryoInputFormat
-gremlin.hadoop.graphOutputFormat=org.apache.hadoop.mapreduce.lib.output.NullOutputFormat
-gremlin.hadoop.inputLocation=./data/grateful-dead.kryo
-gremlin.hadoop.outputLocation=output
-gremlin.hadoop.jarsInDistributedCache=true
+gremlin.hadoop.graphReader=org.apache.tinkerpop.gremlin.hadoop.structure.io.gryo.GryoInputFormat
+gremlin.hadoop.graphWriter=org.apache.hadoop.mapreduce.lib.output.NullOutputFormat
 
-#
-# GiraphGraphComputer Configuration
-#
-giraph.minWorkers=2
-giraph.maxWorkers=2
-giraph.useOutOfCoreGraph=true
-giraph.useOutOfCoreMessages=true
-mapred.map.child.java.opts=-Xmx1024m
-mapred.reduce.child.java.opts=-Xmx1024m
-giraph.numInputThreads=4
-giraph.numComputeThreads=4
-giraph.maxMessagesInMemory=100000
+gremlin.hadoop.jarsInDistributedCache=true
+gremlin.hadoop.inputLocation=hdfs:///grateful-dead.kryo
+gremlin.hadoop.outputLocation=output
+gremlin.spark.persistContext=true
 
 #
 # SparkGraphComputer Configuration
 #
 spark.master=local[*]
-spark.executor.memory=1g
+spark.executor.memory=6g
```

Comment:
Need this 6g?

Reply:
Haha, sorry my bad. Well my applications needed 6g because of huge size. We certainly can reduce that to 1g in docs as this is just for reference.